Tests/SentryTests/Helper/SentryLogTests.swift

```diff
@@ -1,17 +1,27 @@
 import XCTest
 
 class SentryLogTests: XCTestCase {
-    
+    var oldDebug: Bool!
+    var oldLevel: SentryLevel!
+    var oldOutput: SentryLogOutput!
+
+    override func setUp() {
+        super.setUp()
+        oldDebug = SentryLog.isDebug()
+        oldLevel = SentryLog.diagnosticLevel()
+        oldOutput = SentryLog.logOutput()
+    }
+
     override func tearDown() {
         super.tearDown()
-        // Set back to default
-        SentryLog.configure(true, diagnosticLevel: SentryLevel.error)
-        SentryLog.setLogOutput(nil)
+        SentryLog.configure(oldDebug, diagnosticLevel: oldLevel)
+        SentryLog.setLogOutput(oldOutput)
```

Comment:
I set these to the original values instead of nil, which would cut off logs for the remainder of the test suite. Similarly in SentryBaseIntegrationTests.